hubble/utils/auth.py

```diff
@@ -17,64 +17,128 @@
 )
 
 NOTEBOOK_LOGIN_HTML = f"""
-<center>
-    <img src={JINA_LOGO} width=175 alt="Jina AI">
-    <p><br></p>
-    <p>
-        Copy a <b>Personal Access Token</b>, paste it below, and press the Login button.
-        <br>
-        If you do not have a token, press the Login button to login via the browser.
-    </p>
-</center>
-"""
-
-NOTEBOOK_LOGIN_HTML_NOTE = """
-<center>
-    <p><br></p>
-    <p>
-        <i>
-            You can create a <b>Personal Access Token</b> on the
-            <a href='https://hub.jina.ai/user/tokens' target='__blank'>tokens</a>
-            page.
-        </i>
-    </p>
-</center>
+<div class='custom-container'>
+    <style>
+        .button1 {{
+            color: white;
+            background-color: #009191;
+            border: 1px solid #009191;
+        }}
+        .button2 {{
+            color: #009191;
+            background-color: white;
+            border: 1px solid #009191;
+        }}
+        .link1 {{
+            color:#009191;
+            position: relative;
+            top: 22px;
+            right: -120px;
+            z-index: 99;
+        }}
+        .custom-container {{
+            margin-top: 10px;
+            margin-bottom: -10px;
+        }}
+        .spaced {{
+            margin: 20px 0;
+        }}
+    </style>
+    <center>
+        <img src={JINA_LOGO} width=175 alt='Jina AI'>
+        <div class='spaced'></div>
+        <p>
+            Copy a <b>Personal Access Token</b>, paste it below, and press the <b>Login with Token</b> button.
```

Comment:
This should be "Log in". And what case are we using on buttons elsewhere? Should we use capital T on Token?

Reply:
From what I've seen in other places we use **Login**, and I also think it should be **Login**. 
It this context it can be seen as a noun phrase, so it's not necessarily wrong. Plus, I think **Log in with token** is too wordy for a button. 
If you want, we can switch to "Token login" and "Browser login".

<img width="644" alt="Screen Shot 2022-10-14 at 21 09 11" src="https://user-images.githubusercontent.com/15995496/195855390-704c63b5-8900-4ed4-863a-eb89138e5e8f.png">

Reply:
> If you want, we can switch to Token login and Browser login.

I like this